repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorHandler.java

```diff
@@ -153,9 +146,16 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 }
 
                 if (!isValid) {
-                    if (mbcsInvalidRequests != null) {
+                    if (Optional.ofNullable(metricsService).isPresent()) {
```

Comment:
My mistake. I meant just Spring, not Spring Boot.

I'm not sure if we check on other services currently or not. However, since we are moving towards all services being first class citizens of the Repose environment, a user could conceivably turn something off and then there wouldn't be something to pass in.